roles/etcd_ca/tasks/main.yml

```diff
@@ -38,9 +38,9 @@
   delegate_to: "{{ etcd_ca_host }}"
   run_once: true
 
-- command: touch {{ etcd_ca_db }}
-  args:
-    creates: "{{ etcd_ca_db }}"
+- file:
+    path: "{{ etcd_ca_db }}"
+    state: touch
```

Comment:
Touching file for subsequent run will not make any changes though except the last modification date for the file. And the warning I'm trying to avoid in this PR basically `[WARNING]: Consider using file module with state=touch rather than running touch`